### PR TITLE
Rpm packaging

### PR DESCRIPTION
**Pull request by hargoniX**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### Code changes:

* [x] Do all tests pass locally?
    * [x] `tox`
    * [x] `tox -e flake8`
    * [x] `tox -e codespell`
* [ ] Have you updated the [CHANGELOG](CHANGELOG.md)? As i didnt edit code no
* [x] Did you add your name to the contributors list in the [README](README.md)?

### Summary of changes:
* Adding scripts and specfiles for creating RPM packages

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,6 +37,10 @@
         'console_scripts': [
             'getmac = getmac.__main__:main',
         ],
+    } if sys.version_info[:2] <= (2, 7) else {
+        'console_scripts': [
+            'getmac3 = getmac.__main__:main',
```
[the remark this answers is not in the document]
Comment (hargoniX) [reply setup.py]: I've seen other popular tools like for example scapy having a 2.7 and a 3.6 command in the form of scapy and scapy3 too so i do not think there is a way around two executors.

File: README.md [hunk 2]
```diff
@@ -226,6 +226,7 @@ to ask questions or discuss the project (Handle: @KnownError).
 * Jose Gonzalez (@Komish) - Docker container and Docker testing
 * @fortunate-man - Awesome usage videos
 * @martmists - legacy Python compatibility improvements
+* @hargoniX - scripts and specfiles for RPM packaging
```
[the remark this answers is not in the document]
Comment (hargoniX) [reply README.md]: Done.

**Comment by GhostofGoes** [comment]
As an aside, will these spec files work to build RPMs that will work on RHEL/CentOS as well?

**Comment by hargoniX** [comment]
They should work as i am not aware of anything fedora specific I did.

File: python2-getmac.spec [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+BuildArch:      noarch
+BuildRequires:  python2-devel
+
+%description
+Pure-python module to get the MAC address of remote hosts or network interfaces.
+It provides a platform-independent interface to get the MAC addresses of network
+interfaces on the local system(by interface name) and remote hosts on the local
+network (by IPv4/IPv6 address or host name).
+
+%{?python_provide:%python_provide python3-getmac}
```
Comment (zackse) [inline python2-getmac.spec]: Should this be `python2-getmac`?

**Pull request merged**